### PR TITLE
correct doc for DF.count

### PR DESCRIPTION
**Pull request by Rubtsowa**

### CODE REVIEW

File: examples/dataframe/dataframe_count.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+def dataframe_count():
+    df = pd.DataFrame({"A": [.2, .0, .6, .2],
+                       "B": [.5, .6, .7, .8],
+                       "C": [2, 0, 6, 2],
+                       "D": [.2, .1, np.nan, .5],
+                       "E": [-1, np.nan, 1, np.inf],
+                       "F": [np.nan, np.nan, np.inf, np.nan]})
+    out_df = df.count()
+
+    return out_df
```
Comment (densmirn) [inline examples/dataframe/dataframe_count.py]: Let's combine these lines to a single return statement.

File: examples/dataframe/dataframe_count.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+# dtype: int64
+
+import pandas as pd
+import numpy as np
+from numba import njit
+
+
+@njit
+def dataframe_count():
+    df = pd.DataFrame({"A": [.2, .0, .6, .2],
```
Comment (densmirn) [inline examples/dataframe/dataframe_count.py]: I think it's enough to have no more than 3 different columns in DF in the example.

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 3]
```diff
@@ -944,6 +944,40 @@ def prod_overload(df, axis=None, skipna=None, level=None, numeric_only=None, min
...
+        :ref:`Series.count <pandas.Series.count>`
+            Number of non-NA elements in a Series.
+
+        :ref:`DataFrame.shape <pandas.DataFrame.shape>`
+            Number of DataFrame rows and columns (including NA elements).
+
+        :ref:`DataFrame.isna <pandas.DataFrame.isna>`
+            Boolean same-sized DataFrame showing places of NA elements.
+
+
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: Is it required to have 2 empty line instead a single one?

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 4]
```diff
@@ -944,6 +944,40 @@ def prod_overload(df, axis=None, skipna=None, level=None, numeric_only=None, min
...
+    Intel Scalable Dataframe Compiler Developer Guide
+    *************************************************
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: I would insert empty line after asterisks.

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 5]
```diff
@@ -944,6 +944,40 @@ def prod_overload(df, axis=None, skipna=None, level=None, numeric_only=None, min
...
 def count_overload(df, axis=0, level=None, numeric_only=False):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.DataFrame.count
+
+    Limitations
+    -----------
+    Parameters axis, level, numeric_only unsupported.
```
Comment (AlexanderKalistratov) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: are unsupported

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 6]
```diff
@@ -1032,26 +1032,44 @@ def prod_overload(df, axis=None, skipna=None, level=None, numeric_only=None, min
...
+        :ref:`DataFrame.isna <pandas.DataFrame.isna>`
+            Boolean same-sized DataFrame showing places of NA elements.
+
+    Intel Scalable Dataframe Compiler Developer Guide
+    *************************************************
+
+    Pandas DataFrame method :meth:`pandas.DataFrame.count` implementation.
+    .. only:: developer
+
+    Test: python -m sdc.runtests -k sdc.tests.test_dataframe.TestDataFrame.test_count*
```
Comment (PokhodenkoSA) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]:
```suggestion
    .. only:: developer

        Test: python -m sdc.runtests -k sdc.tests.test_dataframe.TestDataFrame.test_count*
```

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 7]
```diff
@@ -1032,26 +1032,44 @@ def prod_overload(df, axis=None, skipna=None, level=None, numeric_only=None, min
...
+            Number of DataFrame rows and columns (including NA elements).
+
+        :ref:`DataFrame.isna <pandas.DataFrame.isna>`
+            Boolean same-sized DataFrame showing places of NA elements.
+
+    Intel Scalable Dataframe Compiler Developer Guide
+    *************************************************
+
+    Pandas DataFrame method :meth:`pandas.DataFrame.count` implementation.
+    .. only:: developer
```
Comment (PokhodenkoSA) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]:
```suggestion

        .. only:: developer
```

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 8]
```diff
@@ -1032,26 +1032,45 @@ def prod_overload(df, axis=None, skipna=None, level=None, numeric_only=None, min
...
 
-    Returns
-    -------
-    :obj:`pandas.Series` or `pandas.DataFrame`
-      for each column/row the number of non-NA/null entries. If level is specified returns a DataFrame.
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/dataframe/dataframe_count.py
+       :language: python
+       :lines: 36-
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: Did you generate the documentation and check the example visually?

File: sdc/datatypes/hpat_pandas_dataframe_functions.py [hunk 9]
```diff
@@ -1032,26 +1032,42 @@ def prod_overload(df, axis=None, skipna=None, level=None, numeric_only=None, min
...
 
-    Returns
-    -------
-    :obj:`pandas.Series` or `pandas.DataFrame`
-      for each column/row the number of non-NA/null entries. If level is specified returns a DataFrame.
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/dataframe/dataframe_count.py
+       :language: python
+       :lines: 38-
```
Comment (densmirn) [inline sdc/datatypes/hpat_pandas_dataframe_functions.py]: Are you sure about the line number ?

**Pull request merged**